index.bs

```diff
@@ -542,6 +542,19 @@ When the <dfn method for="XRFrame">getHitTestResultsForTransientInput(|hitTestSo
 Geometric primitives {#geometric-primitives}
 ====================
 
+XRRayDirectionInit {#xr-ray-direction-init-dictionary}
+--------------------
+
+An {{XRRayDirectionInit}} dictionary represents a direction vector to be passed to the {{XRRay(origin, direction)}} constructor.
+
+<script type="idl">
+dictionary XRRayDirectionInit {
+  double x = 0;
```

Comment:
`unrestricted double` to align with `DOMPointInit` ?

Reply:
Do we want these to be `unrestricted`? I feel like we don't want NaN values here

Reply:
It should be OK to leave as-is, we can always relax this later if need be.